files/default/sql/ddl/3.7.0__initial_tables.sql

```diff
@@ -407,6 +407,24 @@ CREATE TABLE IF NOT EXISTS `feature_group_statistics` (
 
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Table structure for table `feature_group_statistics`
+--
+
+CREATE TABLE IF NOT EXISTS `feature_view_statistics` (
```

Comment:
why is it back?

Reply:
There were two options, either expanding the training_dataset_statistics table (e.g., with window times columns, another unique key, etc) or having the feature_view_statistics table back. I tried the first option but it was getting too complex. Having the feature_view_statistics table back is definitely a better option.

Part of the complexity was also due to the fact that we need to store statistics computed on data obtained from a feature view, but that do not relate to any training dataset (no training dataset version).